integration-tests/modules/__tests__/cart/store/cart.workflows.spec.ts

```diff
@@ -669,4 +675,142 @@ describe("Carts workflows", () => {
       })
     })
   })
+
+  describe("createPaymentCollectionForCart", () => {
+    it("should create a payment collection and link it to cart", async () => {
+      const region = await regionModuleService.create({
+        name: "US",
+        currency_code: "usd",
+      })
+
+      const cart = await cartModuleService.create({
+        currency_code: "usd",
+        region_id: region.id,
+        items: [
+          {
+            quantity: 1,
+            unit_price: 5000,
+            title: "Test item",
+          },
+        ],
+      })
+
+      await createPaymentCollectionForCartWorkflow(appContainer).run({
```

Comment:
Should we only rely on REST API endpoints in the test, considering it's an integration test? Or is it because the endpoints are not yet available?

Reply:
I intentionally split the integration tests into API (`/store/carts.spec.ts`) and Workflows (`/store/cart.workflows.ts`). It didn't feel right to place tests for different inputs and compensation flows of workflows in the API folder.

Reply:
Ah missed the .workflows. part, 👍